aptiblejs/src/cookie.js

```diff
@@ -0,0 +1,26 @@
+function all() {
+  let cookies = {};
+  const cookieStrings = document.cookie.split(';');
+
+  for (let cookie of cookieStrings) {
+    const pieces = cookie.split('=');
+    if (pieces.length !== 2) {
+      continue;
+    }
+
+    cookies[pieces[0].trim()] = pieces[1].trim();
+  }
+
+  return cookies;
+}
+
+export function get(name) {
+  return all()[name];
+}
+
+export function write(name, value, expiresAt) {
+  let cookieString = `${name}=${value}; expires=${expiresAt.toUTCString()};`;
+  cookieString += `domain=.${window.location.hostname.replace('www.', '')}; path=/`;
```

Comment:
Not the biggest fan of assuming the domain is `www.` - can we make this a constant somewhere so it's a little more explicit? Alternatively, check the URL starts with `www.`?